cli.py

```diff
@@ -0,0 +1,166 @@
+import argparse
+from sys import exit
```

Comment:
Minor: the usual usage I've seen is `sys.exit(status_code)`, so you could switch to `import sys` here.